app/models/survey/record.rb

```diff
@@ -10,6 +10,7 @@ class Record < ApplicationRecord
 
     scope :completed, -> { where.not(completed_at: nil) }
     scope :latest_completed, -> { completed.order(completed_at: :desc) }
+    scope :residential_use, -> { completed.where("data->>'has_residential_use' = ?", "true") }
```

Comment:
Maybe worth looking at an index for this:

https://www.postgresql.org/docs/12/datatype-json.html#JSON-INDEXING

Reply:
I've added a comment to keep track of it; don't have enough time to make it happen but it should hold for a while before it reaches the scale to warrant the index.